clang/lib/APINotes/APINotesWriter.cpp

```diff
@@ -1093,6 +1093,7 @@ unsigned getFunctionInfoSize(const FunctionInfo &FI) {
   for (const auto &P : FI.Params)
     size += getParamInfoSize(P);
   size += sizeof(uint16_t) + FI.ResultType.size();
+  size += FI.SwiftReturnOwnership.size();
```

Comment:
Is this correct? We also write the length of the string, so not sure if that is accounted for. This problem would go away if we switched to an enum.